### PR TITLE
[pt1][quant] Add the support of handle Bias being nullptr for torch.ops.quantized.fbgemm_conv

### PR DESCRIPTION
**Pull request by jianyuh**
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#22472 [pt1][quant] Add the support of handle Bias being nullptr for torch.ops.quantized.fbgemm_conv**&nbsp;&nbsp;[:green_heart:](https://our.internmc.facebook.com/intern/diff/D16097594/)

As Title says.

Differential Revision: [D16097594](https://our.internmc.facebook.com/intern/diff/D16097594/)

### CODE REVIEW

File: aten/src/ATen/native/quantized/cpu/qconv.cpp [hunk 1]
```diff
@@ -133,9 +134,16 @@ class QConv2dInt8 final : public c10::OperatorKernel {
 
     fbgemm::DoNothing<> NoOpObj{};
 
-    auto bias_contig = bias.contiguous();
-    const auto* bias_ptr =
-        reinterpret_cast<int32_t*>(bias_contig.data<c10::qint32>());
+    const int32_t* bias_ptr = nullptr;
+    if (bias.has_value()) {
+      Tensor bias_vec = bias.value_or(Tensor());
```
Comment (bddppq) [inline aten/src/ATen/native/quantized/cpu/qconv.cpp]: Here bias for sure has value, so you can directly call `bias.value()`

**Review by bddppq — COMMENTED** [review]
Add a test case?

File: aten/src/ATen/native/quantized/cpu/qconv.cpp (hunk 1 above)
Comment (jianyuh) [reply aten/src/ATen/native/quantized/cpu/qconv.cpp]: Fixed. Thanks!

**Comment by jianyuh** [comment]
Will add a test case.

**Comment by jianyuh** [comment]
> Add a test case?

Thanks @bddppq for the great help! Added the test case for ```b == None```.

**Review by bddppq — APPROVED** [review]
LG!

**Pull request closed without merging**